web/thesauruses/typescript/strings.json

```diff
@@ -0,0 +1,338 @@
+{
+  "meta": {
+		"language": "ts",
```

Comment:
This should be "typescript" to match the conventions used elsewhere in CT.

Reply:
Whoops, changed.